hack/e2e-suite/guestbook.sh

```diff
@@ -38,15 +38,22 @@ GUESTBOOK="${KUBE_ROOT}/examples/guestbook"
 function teardown() {
   ${KUBECTL} stop -f "${GUESTBOOK}"
   if [[ "${KUBERNETES_PROVIDER}" == "gce" ]]; then
-    gcloud compute forwarding-rules delete "${INSTANCE_PREFIX}-default-frontend" || true
-    gcloud compute target-pools delete "${INSTANCE_PREFIX}-default-frontend" || true
-  fi  
+    gcloud compute forwarding-rules delete -q "${INSTANCE_PREFIX}-default-frontend" || true
+    gcloud compute target-pools delete -q "${INSTANCE_PREFIX}-default-frontend" || true
+  fi
 }
 
 prepare-e2e
 
 trap "teardown" EXIT
 
+if gcloud config list compute/region --format text | grep 'compute.region: None'; then
```

Comment:
Doesn't that just take the ZONE from an env variable and use it to recreate the region, are you recommending a similar approach? 

I could use sed/awk or something to parse out 'None' but I figured eitherway I won't know what to search for, and if the format of the string changes both will most likely break. Thoughts?